web/src/components/newsCard.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Img from 'gatsby-image';
+
+export default function NewsCard({ data }) {
+    const {
+        title,
+        description,
+        image: {
+            childImageSharp: { fluid },
+        },
+    } = data;
+
+    return (
+        <div className="bg-white h-[34rem] w-80">
```

Comment:
This is a very specific height. Have you tried it with longer/shorter content than is in the designs?

Reply:
@asasmith text will be hidden if it goes over, I can apply an overflow scroll class. Does that work for you? Looks good with less text.

Reply:
@asasmith https://deploy-preview-18--airtec.netlify.app/

Reply:
I think this is fine for now. Let's keep an eye on it as real content is being added.